docs/demos/tutorials/07_Quality_assurance.ipynb

```diff
@@ -5494,6 +5494,40 @@
     "roc_table = linker.truth_space_table_from_labels_table(labels_table)\n",
     "roc_table.as_pandas_dataframe(limit=5)"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ff86458e",
+   "metadata": {},
+   "source": []
```

Comment:
The truth table may need to have its width limited. It runs the full length of my screen and partially obscures the right sidebar.

Reply:
I'm not sure if there is something we can fix easily. It happens in a few different places across the examples and tutorials and I had kinda just accepted it. But happy to revisit if you think it is worthwhile!